### DISCUSSION TITLE
How to pass cli argument?

### DISCUSSION THREAD

**Discussion by avi-testsigma** [crxjs/chrome-extension-tools#879] [Q&A] [answered]
Hi.. I want to use `defineManifest` to load separate config for `chrome` and `firefox`. How can I achieve this? I don't want to use .env because that has a different purpose, different env file for different environments. But chrome and firefox are like target and I should be able to control them from package.json

**Comment by avi-testsigma** [comment, marked as answer]
Got it. We can just use process.env

**Reply by farzadmf** [reply]
Hey @avi-testsigma , by any chance, do you have an example building an extension for both Firefox and Chrome (with dev mode and hot reloading for both at the same time)?

Things are working with Chrome, but I have a hard-time making things work for Firefox

**Reply by avi-testsigma** [reply]
You are correct @farzadmf . It is not working in dev mode. I had verified it in build. It works fine there. In dev mode Firefox is not allowing to connect to http://localhost:5173 as it breaks CSP rules
